onmt/trainer.py

```diff
@@ -324,8 +329,7 @@ def validate(self, valid_iter, moving_average=None):
         # Set model in validating mode.
         valid_model.eval()
 
-        sources = []
-        refs = []
+        tokenized_batchs = []
```

Comment:
use batches instead of batchs.
But is the point only "tokenization" or full "transform" ?